File: C/#1001 - #1050/#1012-Area.c
//Area
#include <stdio.h>

int main(){
	double A,B,C;
	double pi = 3.14159;  

	scanf("%lf %lf %lf",&A,&B,&C);

	printf("TRIANGULO: %.3lf\n",(A*C)/2);
	printf("CIRCULO: %.3lf\n",(C*C*pi));
	printf("TRAPEZIO: %.3lf\n",((A+B)*C)/2);
	printf("QUADRADO: %.3lf\n",(B*B));
	printf("RETANGULO: %.3lf\n",(A*B));
		
		return 0;
}

File: Iniciante-Begginer/Python/#1001 - #1050/#1012-Area.py
#Author: Jocelino F.G


A,B,C= list(map(float, input().split()))

pi = 3.14159

tri = (A*C)/2
cir = (C*C*pi)
tra = ((A+B)*C)/2
qua = (B*B)
ret = (A*B)

print("TRIANGULO: %0.3f" % tri)
print("CIRCULO: %0.3f" % cir)
print("TRAPEZIO: %0.3f" % tra)
print("QUADRADO: %0.3f" % qua)
print("RETANGULO: %0.3f" % ret)